lib/safari/index.js

```diff
@@ -113,13 +120,13 @@ Safari.prototype.authorizeUser = function authorizeUser(username, password) {
 */
 Safari.prototype.fetchResource = function fetchResource(url, options) {
   debug(`fetchResource called with URL: ${url}`);
-  if(!url || !this.accessToken) return Promise.reject("url was not specified or user has not been authorized yet");
+  if(!url || !this.accessToken && !this.cookieString) return Promise.reject("url was not specified or user has not been authorized yet");
   // ## prepare options for resource request
   var uri = `${this.baseUrl}/${url}`;
```

Comment:
For some reason, either the request lib or the api cannot handle double slashes
```suggestion
  var uri = `${this.baseUrl}${url}`;
```